src/main/java/lotto/util/LottoGenerator.java

```diff
@@ -0,0 +1,30 @@
+package lotto.util;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoGenerator {
+
+  private static final int MIN_LOTTO_NUMBER = 1;
+  private static final int MAX_LOTTO_NUMBER = 45;
+  private static final int MIN_LOTTO_INDEX = 0;
+  private static final int MAX_LOTTO_INDEX = 6;
```

Comment:
```suggestion
  private static final int LOTTO_SIZE = 6;
```
이렇게 하는게 의미가 명확할것 같은데 어떤가요?🤔

Reply:
좋은 것 같은데 . 
한가지 쫌 찜찜한 것이 
`  private static final int MIN_LOTTO_INDEX = 0;
`
는 다른 사람이 보기에 헷갈릴 수도 있겠단 생각이 드네요.. 그래서 통일은 하긴 했었는데..  어떻게 생각하시나요?

우선 수정하였습니다.

Reply:
MIN_LOTTO_INDEX는 ZERO_INDEX 정도로 해볼까요?🤔